xml/mysql_banners.xml

```diff
@@ -175,6 +186,8 @@
     <param pos="0" name="os.family" value="Linux"/>
     <param pos="0" name="os.product" value="Linux"/>
     <param pos="0" name="os.version" value="12.04"/>
+    <param pos="0" name="os.certainty" value="0.80"/>
+    <param pos="0" name="os.device" value="General"/>
```

Comment:
IMO, we should only set `os.device` to something if it is unique in some way -- `General` is too generic.